lib/Chat/Changelog/Manager.php

```diff
@@ -102,6 +112,10 @@ public function getChangelogs(): array {
 			$this->l->t('- Check out the new grid and call view'),
 			$this->l->t('- You can now upload and drag\'n\'drop files directly from your device into the chat'),
 			$this->l->t('- Shared files are now opened directly inside the chat view with the viewer apps'),
+			$this->l->t('New in Talk 10'),
+			$this->l->t('- You can now search for chats and messages in the unified search in the top bar'),
+			$this->l->t('- Spice up your messages with emojis from the emoji picker') . $emojis,
```

Comment:
The emoji should be included in the `l->t()` string for proper localisation (for example, for right-to-left languages).

Reply:
I fear they break transifex :-X

Reply:
Solved